framework/sandstone.cpp

```diff
@@ -3032,16 +3039,24 @@ int internal_main(int argc, char **argv)
             } else if (strncasecmp(optarg, "priority", 3) == 0) {
                 sApp->test_selection_strategy = Prioritized;
             } else {
-                fprintf(stderr, "Cannot determine weighted testrunner type (%s is invalid - use repeat/non-repeat)", optarg);
+                fprintf(stderr, "%s: Cannot determine weighted testrunner type (%s is invalid - use repeat/non-repeat)", argv[0], optarg);
                 return EX_USAGE;
             }
             sApp->weighted_testrunner_runtimes = NormalTestrunTimes;
             break;
 
         case test_list_file_option:
+            if ((optarg == NULL) || (*optarg == '\0')) {
```

Comment:
It can't be null because it's a `required_argument`, but it's no harm to check here.

Reply:
Removed to not introduce more "not very harmfull" code.